jobs/Backend/Task/test/Mews.ExchangeRateProvider.Tests/ExchangeRateProviderPolicyDecoratorTests.cs

```diff
@@ -0,0 +1,40 @@
+using Moq;
+using Polly;
+using Polly.Registry;
+
+namespace Mews.ExchangeRateProvider.Tests;
+
+[TestFixture]
+public sealed class ExchangeRateProviderPolicyDecoratorTests
+{
+    private Mock<IExchangeRateProvider> _innerMock;
+    private Mock<ResiliencePipelineProvider<string>> _policyMock;
+
+    [SetUp]
+    public void SetUp()
+    {
+        _innerMock = new Mock<IExchangeRateProvider>();
+
+        _innerMock
+            .Setup(erp => erp.GetExchangeRatesAsync(It.IsAny<IEnumerable<Currency>>(), It.IsAny<CancellationToken>()))
+            .ReturnsAsync(Enumerable.Empty<ExchangeRate>());
+
+        _policyMock = new Mock<ResiliencePipelineProvider<string>>();
+
+        _policyMock
+            .Setup(p => p.GetPipeline("ExchangeRateProviderPolicyDecorator"))
+            .Returns(ResiliencePipeline.Empty);
+    }
+
+    [Test]
+    public async Task Forwards_request_to_inner_policy()
```

Comment:
This is a fairly basic test of the decorator class in isolation.  It isn't testing the Polly policy properly - to do that we would need to set up an integration testing project which would typically be associated with a library consumer.